docs/example_compartmental_elongated_cells_tutorial.rst

```diff
@@ -0,0 +1,805 @@
+
+
+Curvature Plugin
+++++++++++++++++
+
+Let us now put everything together an implement elongated compartmentalized cell. The solution that will prevent two
+``Center`` cells (the ones that initially were touching ``Top`` cell), from forming an extra FPP link,
+is to use Curvature Plugin. The way Curvature plugin works is by constraining the angle that two adjacent links can form
+By using high value of Curvature lambda you may constrain two adjacent links to form a straight line
+and by adiabatically lowering the lambda you can control how much elongated cell can bend.
+The code for this section is in ``Demos/CompuCellPythonTutorial/ElongatedCellsTutorial/Tutorial_08``
+
+HEre is the code that we add to the XML to enable Curvature energy calculations:
+
+.. code-block:: XML
+
+    <Plugin Name="Curvature">
+
+        <InternalParameters Type1="Top" Type2="Center">
+            <Lambda>1000.0</Lambda>
+            <ActivationEnergy>-50.0</ActivationEnergy>
+        </InternalParameters>
+
+        <InternalParameters Type1="Center" Type2="Center">
+            <Lambda>1000.0</Lambda>
+            <ActivationEnergy>-50.0</ActivationEnergy>
+        </InternalParameters>
+
+
+
+        <InternalTypeSpecificParameters>
+            <Parameters TypeName="Top" MaxNumberOfJunctions="1" NeighborOrder="1"/>
+            <Parameters TypeName="Center" MaxNumberOfJunctions="2" NeighborOrder="1"/>
+        </InternalTypeSpecificParameters>
+
+    </Plugin>
+
+
+With this extra addition the compartments will form a line even if we let the simulation run for a very long time:
+
+|img013|
+
+
+As you probably have noticed, the syntax for this plugin resembles the syntax of the FPP plugin - we have ``<Lambda>``,
+``<ActivationEnergy>``, ``MaxNumberOfJunctions`` . This is because Curvature plugin establishes its own set of "links" between
+cells but those links are not used to penalize intercellular distance but rather to penalize the deviation from straight line
+arrangement of compartment cells
+
+Adding persistent motion to cells
++++++++++++++++++++++++++++++++++
+
+Let us add a bit more code to make this simulation more interesting. First, we will create more cells. We will use
+our convenience function ``create_arranged_cells`` and as a result all of those cells will be arranged vertically -
+this will not be a problem though because, next, we will be applying random force to the "first" cell of each cluster i.e.
+to the cell that is created first in each cluster. We will store a list of "first" cells inside member variable
+``self.list_of_leading_cells = []`` which is a list. . we have to be careful to ensure that cells stored in that list do
+not disappear because if the do disappear and we try to reference them we will get Segmentation Fault Error.
+We will show later how we could avoid this issue in the code , just to show you how to handle situation of that type.
```

Comment:
To help break up this long paragraph, move 
> "we have to be careful to ensure that cells stored in that list do
not disappear because if the do disappear and we try to reference them we will get Segmentation Fault Error.
We will show later how we could avoid this issue in the code , just to show you how to handle situation of that type."

a little ways down so that it is closer to the code that addresses this pitfall.